addons/event/models/event_registration.py

```diff
@@ -30,6 +30,11 @@ def _get_random_barcode(self):
          """
         return str(int.from_bytes(os.urandom(8), 'little'))
 
+    def _get_registration_summary(self):
```

Comment:
Method is defined in this file, hence can be integrated directly in it :)

Reply:
oops